linetools/isgm/utils.py

```diff
@@ -325,6 +326,55 @@ def complist_from_table(table):
     return complist
 
 
+def table_from_complist(complist):
+    """
+    Returns a QTable from an input list of AbsComponents. It only
```

Comment:
I have to admit that I'm no longer too enamored with QTable's.  I was at the start but find that normal Table objects do 99% of what is needed.

Do we need one here?